samples/nats_jetstream/v3/message-interoperability/main.go

```diff
@@ -0,0 +1,78 @@
+/*
+ Copyright 2021 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"context"
+	"io"
+	"log"
+	"os"
+
+	"github.com/kelseyhightower/envconfig"
+
+	cloudeventsjsm "github.com/cloudevents/sdk-go/protocol/nats_jetstream/v3"
+	cloudeventshttp "github.com/cloudevents/sdk-go/v2/protocol/http"
+)
+
+type envConfig struct {
+	// Port on which to listen for cloudevents
+	Port int `envconfig:"PORT" default:"8080"`
+
+	// NATSServer URL to connect to the nats server.
+	NATSServer string `envconfig:"NATS_SERVER" default:"http://localhost:4222" required:"true"`
+
+	// Subject is the nats subject to publish cloudevents on.
+	Subject string `envconfig:"SUBJECT" default:"sample" required:"true"`
+}
+
+func main() {
+	var env envConfig
+	if err := envconfig.Process("", &env); err != nil {
+		log.Printf("[ERROR] Failed to process env var: %s", err)
+		os.Exit(1)
+	}
+	ctx := context.Background()
+
+	natsProtocol, err := cloudeventsjsm.NewSender(ctx, env.NATSServer, env.Subject, cloudeventsjsm.NatsOptions(), nil)
+	if err != nil {
+		log.Fatalf("failed to create nats protcol, %s", err.Error())
+	}
+
+	defer natsProtocol.Close(ctx)
+
+	httpProtocol, err := cloudeventshttp.New(cloudeventshttp.WithPort(env.Port))
```

Comment:
What is this interoperability test asserting? Not sure this is needed here because it's an example. Consider removing or moving to integration test